charts/images.yaml

```
@@ -59,3 +59,8 @@ images:
   sourceRepository: github.com/kubernetes-csi/livenessprobe
   repository: quay.io/k8scsi/livenessprobe
   tag: "v2.0.0"
+
+- name: remedy-controller-azure
+  sourceRepository: github.wdf.sap.corp/kubernetes/remedy-controller
```

Needs an update now:
```suggestion
  sourceRepository: github.com/gardener/remedy-controller
```